README.md

```diff
@@ -1,14 +1,62 @@
-# How to Add the Syncfusion Blazor Button Component to a Blazor Server App  
+# How to Add the Syncfusion Blazor Button Component to a Blazor Server App
 
-Learn how easily you can create and configure Syncfusion Blazor Button component in a Blazor Server App using Visual Studio 2019. In this video, you will learn how to add a simple Blazor Button to a Blazor server app. You will see how to add icons to Blazor Buttons and change the icon positions. Also, you will learn how to toggle the button state as well as how to add predefined styles to Blazor Button. Finally, you will see how to bind a click event to this Blazor Button component.
- 
-Example: https://blazor.syncfusion.com/demos/buttons/default-functionalities
-
-Documentation: https://blazor.syncfusion.com/documentation/button/getting-started
+The [Blazor Buttons Component](https://www.syncfusion.com/blazor-components/blazor-button?utm_source=github&utm_medium=listing&utm_campaign=blazor-button-github-samples) is a custom HTML5 button control. It has several built-in features such as support for icons, predefined styles, different button types, different button sizes, and UI customization. Learn how easily you can create and configure Syncfusion Blazor Button component in a Blazor Server App using Visual Studio 2019. In this video, you will learn how to add a simple Blazor Button to a Blazor server app. You will see how to add icons to Blazor Buttons and change the icon positions. Also, you will learn how to toggle the button state as well as how to add predefined styles to Blazor Button. Finally, you will see how to bind a click event to this Blazor Button component.
 
 ## Project pre-requisites
 Make sure that you have the compatible versions of Visual Studio 2019 and .NET Core SDK latest version(3.1.2) in your machine before starting to work on this project.
 
 ## How to run this application?
 To run this application, you need to first clone the create-blazor-button-components-in-blazor-server-app repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
 
+## Features and Benefits 
+
+## Action or link button
+
+The Blazor Button Component lets you represent a hyperlink graphically using either an icon or text.
+
+## Icon button
+
+Visual information is often processed by users more quickly than verbal information. To accomplish this, an icon-only button or one with an icon to the right or left of the button text has been designed.
+
+## Button size
+
+You have the option of choosing between the default and small button sizes depending on the requirements of your application.
+
+## Related links
+
+[Learn More about Button ](https://www.syncfusion.com/blazor-components/blazor-button?utm_source=github&utm_medium=listing&utm_campaign=blazor-button-github-samples)
```

Comment:
remove the extra space
![image](https://user-images.githubusercontent.com/61148953/176461238-4cda2717-a3e5-447a-a476-04660f8deded.png)